### PR TITLE
#57 Create a TopBar component

### LINKED ISSUE #57: Create a top bar component
image

### PR DESCRIPTION
**Pull request by Koushith**
## Description
- [x] Added TopBar component.

## Screenshot

Mobile View
image

_

## Platform tested
- [x] Web
- [x] Tablet
- [x] Mobile

### CODE REVIEW

File: src/components/primitive/top-bar/top-bar.component.styles.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import { Box, Avatar } from '../../primitive';
```
Comment (koshikraj) [inline src/components/primitive/top-bar/top-bar.component.styles.ts:2]: We can do absolute imports now, right?

File: src/components/primitive/top-bar/top-bar.component.tsx [hunk 2]
```diff
@@ -47,7 +9,7 @@ export const TopBar = () => {
         <Image src={safientLogo} width={12.1} />
       </LogoWrapper>
       <BoxGroup row>
-        <NotificationIcon name='notification' size='medium' />
+        <NotificationIcon name='notification' size='small' />
         <Avatar src='https://pbs.twimg.com/profile_images/1428345850942480393/l84K68AT_400x400.png' size='small' />
```
Comment (koshikraj) [inline src/components/primitive/top-bar/top-bar.component.tsx]: Use a local image

Comment (Koushith) [reply src/components/primitive/top-bar/top-bar.component.tsx]: okay, that was a placeholder image. will update it.

File: src/components/primitive/top-bar/top-bar.component.styles.ts (hunk 1 above)
Comment (Koushith) [reply src/components/primitive/top-bar/top-bar.component.styles.ts:2]: yeah, we can use absolute import. will fix in next commit.

**Review by koshikraj — APPROVED** [review]

**Pull request merged**